cookbook/how_to_create_and_customize_application_menu.rst

```diff
@@ -43,42 +41,50 @@ First you have to go to your default controller, create your action and specify
          */
         public function indexAction()
         {
-            return array('name' => "hello link");
+            return array('name' => "John Doe");
         }
     }
 
+.. seealso::
 
-Next you should specify action template (file path is src/Acme/Bundle/NewBundle/Resources/views/Default/index.html.twig)
-with the following content:
+    You can read more about both the ``@Route`` and the ``@Template`` annotation in the
+    `SensioFrameworkExtraBundle documentation`_.
+
+    .. _`SensioFrameworkExtraBundle documentation`: http://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/index.html
```

Comment:
I would move this line to the end of the file to maintain all external references at one location.